common/internal_model/src/test/scala/uk/ac/wellcome/models/work/generators/WorkGenerators.scala

```diff
@@ -59,9 +70,6 @@ trait WorkGenerators extends IdentifiersGenerators {
   def denormalisedWorks(n: Int): List[Work.Visible[Denormalised]] =
     (1 to n).map(_ => denormalisedWork()).toList
 
-  def identifiedWorks(n: Int): List[Work.Visible[Identified]] =
-    (1 to n).map(_ => identifiedWork()).toList
```

Comment:
Any reason in particular for removing this?

Reply:
It's not used anywhere, AFAICT? I think I thought I was looking at LegacyWorkGenerators when I ran "Find Usages", oops.

Reply:
I'm not sure it is used yet, might be required to replicate the original functionality, I dunno (`sourceWorks` was required for example).

Reply:
Yup, I think it's useful and I'll put it back.